README.md

```diff
@@ -1,7 +1,286 @@
 # HashiCorp Developer - developer.hashicorp.com
 
-Welcome to HashiCorp Developer! This is the home for HashiCorp product reference documentation, along with all other kinds of content for our practitioners.
+Welcome to HashiCorp Developer! This is the home for HashiCorp product reference documentation and tutorials for our practitioners. For background information on this project, refer to [[MKTG-034]](https://docs.google.com/document/d/1ASyBOCWWP8VUahbL5c5y0qrDMgqhYdXJ2h15xzh3JtA/edit#heading=h.spiwwyows3cr).
 
-## Contributing to the project
+## Table of contents
 
-Additional documentation for internal collaborators is located in the [docs directory](./docs/README.md).
+- [Overview](#overview)
+- [Local Development](#local-development)
+- [Accessibility](#accessibility)
+- [Testing](#testing)
+- [Helpers](#helpers)
+- [Component Organziation](#component-organization)
+- [Configuration](#configuration)
+- [Analytics](#analytics)
+- [SEO metadata](#seo-metadata)
+- [Performance](#performance)
+- [Remote Content & Application context](#remote-content--application-context)
+- [Proxied Redirects](#proxied-redirects)
+
+> **Content Authors** Please see [this documentation](./src/content/README.md) for contributing content updates to Developer. Reach out in [#proj-dev-portal](https://hashicorp.slack.com/archives/C01KCU4HDPY) on Slack if you have any issues / questions.
+
+## Overview
+
+- overeall how it works
+- the related backend systems
```

Comment:
Wasn't sure, but seems like these were rough notes that you've since expanded on!

```suggestion
```

Reply:
oops! ty haha